src/tokenizer.ts

```diff
@@ -1697,6 +1697,18 @@ export class Tokenizer extends DiagnosticEmitter {
     return this.readHexadecimalEscape(4, startIfTaggedTemplate);
   }
 
+  hasNoMoreExpressionInLine(): bool {
```

Comment:
Technically not looking for an expression, but for non-whitespace. Not sure what would be a good name, hmm.